client/homebrew/pages/basePages/listPage/listPage.jsx

```diff
@@ -79,15 +81,15 @@ const ListPage = createClass({
 	},
 
 	renderBrews : function(brews){
-		if(!brews || !brews.length) return <div className='noBrews'>No Brews.</div>;
+		if(!brews || !brews.length) return <div className='noBrews'>{'No Brews'.translate()}.</div>;
```

Comment:
This should have `'No Brews'.translate(['userPage'])`.

I'm not leaving these comments in any particular order-- i realize now that I probably should have made all of the comments as part of a "Review", rather than some as part of a review and some as individual comments.  Sorry about that.